src/fios.h

```diff
@@ -178,19 +178,19 @@ class FileList {
 	/** Remove all items from the list. */
 	inline void Clear()
 	{
-		this->files.Clear();
+		this->files.clear();
 	}
 
 	/** Compact the list down to the smallest block size boundary. */
 	inline void Compact()
 	{
-		this->files.Compact();
+		this->files.shrink_to_fit();
```

Comment:
Use the "swap-trick" to force the shrink.
```suggestion
		decltype(files)(this->files).swap(this->files);
```